src/app/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -28,70 +28,26 @@ const Breadcrumbs = ({ query, type, title, url }) => {
 
   if (type === 'item') {
     crumbs = (
-      <span>
-        <Link to="https://nypl.org">Home</Link> &gt;&nbsp;
-        <Link to="https://nypl.org/research">Research</Link> &gt;&nbsp;
-        <Link to="/">Research Catalog</Link> &gt;&nbsp;
-        {
-          query ?
-          (<span>
-            <Link
-              title={`Make a new search for ${query}`}
-              to={`/search?q=${query}`}
-            >Items</Link> &gt;&nbsp;</span>)
-          : null
-        }
-        <span className="currentPage">{currentPageText}</span>
-      </span>
-    );
-  }
-
-  if (type === 'hold') {
-    crumbs = (
-      <span>
-        <Link to="https://nypl.org">Home</Link> &gt;&nbsp;
-        <Link to="https://nypl.org/research">Research</Link> &gt;&nbsp;
-        <Link to="/">Research Catalog</Link> &gt;&nbsp;
+      <ol className="nypl-breadcrumbs">
+        <li><a href="https://nypl.org">Home</a></li>
+        <li><a href="https://nypl.org/research">Research</a></li>
+        <li><Link to="/">Research Catalog</Link></li>
         {
           query ?
-          (<span>
+          (<li>
             <Link
               title={`Make a new search for ${query}`}
               to={`/search?q=${query}`}
-            >Items</Link> &gt;&nbsp;</span>)
+            >Items</Link></li>)
           : null
         }
-        <Link to={`/item/${url}`}>{currentPageText}</Link> &gt;&nbsp;
-        <span className="currentPage">Place a hold</span>
-      </span>
-    );
-  }
-
-  if (type === 'holdConfirmation') {
-    crumbs = (
-      <span>
-        <Link to="https://nypl.org">Home</Link> &gt;&nbsp;
-        <Link to="https://nypl.org/research">Research</Link> &gt;&nbsp;
-        <Link to="/">Research Catalog</Link> &gt;&nbsp;
-        {
-          query ?
-          (<span>
-            <Link
-              title={`Make a new search for ${query}`}
-              to={`/search?q=${query}`}
-            >Items</Link> &gt;&nbsp;</span>)
-          : null
-        }
-        <Link to={`/item/${url}`}>{currentPageText}</Link> &gt;&nbsp;
-        <span className="currentPage">Hold confirmation</span>
-      </span>
+        <li>{currentPageText}</li>
+      </ol>
     );
   }
 
   return (
-    <ol className="nypl-breadcrumbs">
-      {crumbs}
-    </ol>
+    <span>{crumbs}</span>
```

Comment:
Since you're just returning the variable to be rendered, it does not need to be wrapped in any elements. You can just do `return (crumbs);`.